Makefile

```diff
@@ -53,9 +53,12 @@ install:
 check: pytest gstest doctest
 
 
+#: Remove Cython-derived files
+clean-cython:
+	find mathics -name "*.so" -type f -delete
```

Comment:
shouldn't also clean .c files?

Reply:
I am not sure it not strictly necessary, but I guess it is better to be safe here. And this will work when there are no `.so` s but something else...